### PR TITLE
remove .g8 seed, use 00 source download

### PR DESCRIPTION
**Pull request by patriknw**
* created 00-shopping-analytics-service-scala and 00-shopping-order-service-scala
  as if they were generated by the seed template
* add more to copy-identical-files.sh for the new 00
* include 00-shopping-analytics-service-scala and 00-shopping-order-service-scala
  in source downloads
* adjust documentation
* add 00 in .travis.yml

References #305

I will make similar for Java but didn't want to interfere with ongoing work there yet.

### CODE REVIEW

File: scripts/prepare-downloads.sh [hunk 1]
```diff
@@ -134,4 +145,4 @@ prepareTemporalFolder
 fetchProject shopping-cart-service-java shopping-cart-service
 fetchProject shopping-analytics-service-java shopping-analytics-service
 fetchProject shopping-order-service-java shopping-order-service-java
```
Comment (octonato) [inline scripts/prepare-downloads.sh:147]: This one is added -java. It was already like that, but we can fix it while here.

File: scripts/prepare-downloads.sh [hunk 2]
```diff
@@ -58,35 +58,46 @@ removeTags
...
 fetchProject 04-shopping-cart-service-scala shopping-cart-service
+fetchProject 00-shopping-analytics-service-scala shopping-analytics-service
+fetchProject 00-shopping-order-service-scala shopping-order-service-scala
 zipAndAttach 4-shopping-cart-projection-scala.zip
 
 ## projection kafka
 prepareTemporalFolder
 fetchProject 05-shopping-cart-service-scala shopping-cart-service
 fetchProject shopping-analytics-service-scala shopping-analytics-service
+fetchProject 00-shopping-order-service-scala shopping-order-service-scala
```
[the remark this answers is not in the document]
Comment (octonato) [reply scripts/prepare-downloads.sh:100]: I can't comment on it (not on diff), but the last one. Is also adding a `-scala` to the last one. That's probably the origin of all the above. 

See all last one in the java zip. Same mistake, copy-n-pasted the scala one and `s/scala/java/`

**Review by octonato — APPROVED** [review]
Good we have that. Simplifies the instructions.

File: scripts/prepare-downloads.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (patriknw) [reply scripts/prepare-downloads.sh]: thanks, I have fixed all, including -java

**Comment by patriknw** [comment]
merging this, and will follow up with corresponding for Java when the time is right

**Pull request merged**